plugins/forc-postgres/src/ops/forc_postgres_startdb.rs

```diff
@@ -0,0 +1,51 @@
+use crate::{
+    cli::StartDbCommand,
+    utils::{into_postgres_version, load_pg_config},
+};
+use pg_embed::{pg_fetch::PgFetchSettings, postgres::PgEmbed};
+use tracing::info;
+
+pub fn run_forever() {
+    loop {
+        std::thread::sleep(std::time::Duration::from_secs(1));
+    }
+}
+
+pub async fn init(command: StartDbCommand) -> anyhow::Result<()> {
+    let StartDbCommand {
+        name,
+        database_dir,
+        config,
+    } = command;
+
+    let (_database_dir, pgconfig) =
+        load_pg_config(database_dir.as_ref(), config.as_ref(), &name)?;
+
+    let fetch_settings = PgFetchSettings {
+        version: into_postgres_version(&pgconfig.postgres_version),
+        ..Default::default()
+    };
+
+    let mut pg = PgEmbed::new(pgconfig.into(), fetch_settings).await?;
+
+    info!("\nStarting PostgreSQL.\n");
+    pg.start_db().await?;
+
+    let pg_db_uri = pg.full_db_uri(&name);
+
+    match pg.database_exists(&name).await {
+        Ok(exists) => {
+            if exists {
+                info!("✅ Successfully started database at '{pg_db_uri}'.");
+                run_forever();
```

Comment:
In the updated README, we say that a user can Ctrl-C and the database will still be running. If that's the case, do we need to run a loop and keep this particular process alive?

Reply:
I would assume that `forc index postgres stop` would stop all processes, even if I didn't `Ctrl-C` `start postgres`. 

It would be great if `postgres stop` would stop all processes even if `start postgres` was still running.

Or alternatively, if `start postgres` could by itself `Ctrl-C` in the CLI, and continue running in the background. If I'm not mistaken this is similar to how `brew services start/stop postgresql` works. This parity to brew services would be neat.

Reply:
@rvmelkonian I agree. Let me play around with this a bit more.